hw/ip/csrng/doc/_index.md

```diff
@@ -473,27 +472,28 @@ The following waveform shows an example of how the entropy source hardware inter
 
 {{< wavejson >}}
 {signal: [
-   {name: 'clk'       , wave: 'p...|...|....|......'},
-   {name: 'es_enable' , wave: '01..|...|....|......'},
-   {name: 'es_req'    , wave: '0..1|..0|.1..|..0...'},
-   {name: 'es_ack'    , wave: '0...|.10|.10.|.10...'},
-   {name: 'es_bits'   , wave: 'x...|.3x|x3x.|.3x...', data: ['es0','es1','es2','es3','es4','es5','es6','es7']},
-   {name: 'es_fips'   , wave: '0...|...|.10.|.10...'},
- {},
+   {name: 'clk'           , wave: 'p...|.........|.......'},
+   {name: 'es_req'        , wave: '0..1|..01.0..1|.....0.'},
+   {name: 'es_ack'        , wave: '0...|.10.10...|....10.'},
+   {name: 'es_bus[383:0]' , wave: '0...|.30.30...|....30.', data: ['es0','es1','es2']},
+   {name: 'es_fips'       , wave: '0...|....10...|....10.'},
+]}
 ]}
 {{< /wavejson >}}
 
 ### Interrupts
 
-The CSRNG module has two interrupts: `cs_cmd_req_done` and `cs_fifo_err`.
+The CSRNG module has four interrupts: `cs_cmd_req_done` and `cs_fifo_err`.
```

Comment:
I just would remove the signal names from this line so you don't have to keep them in sync.  You list them out one-by-one immediately below and I think we all can count to 4 :)

Reply:
I agree - removing this line.